docs/en/admins/16_OpenID-Connect.md

```diff
@@ -4,7 +4,8 @@ See: [What is OpenID Connect?](https://openid.net/connect/).
 
 This is one of the [access control methods](09_AccessControl.md) supported by FreshRSS.
 
-OIDC support is provided by [mod_auth_openidc](https://github.com/OpenIDC/mod_auth_openidc).
+OIDC support is provided by the [mod_auth_openidc](https://github.com/OpenIDC/mod_auth_openidc)
+Apache module. OIDC is not supported on web servers other than Apache.
```

Comment:
The formulation is not exactly true. OIDC in FreshRSS is supported by any Web server that implements an OIDC module. I have not investigated whether any Web server implements it though, besides Apache.

Reply:
Indeed, I would remove that last sentence or rephrase it as something like"this documentation only covers..."

Reply:
@Frenzie What about https://github.com/FreshRSS/FreshRSS/pull/7135#discussion_r1895586597 ?

Reply:
That works.